fastapi_opa/auth/auth_saml.py

```diff
@@ -29,15 +40,65 @@ async def authenticate(
         auth = await self.init_saml_auth(request_args)
 
         if "acs" in request.query_params:
+            print(datetime.utcnow(), '--acs--')
             return await self.assertion_consumer_service(auth, request_args)
         # potentially extend with logout here
+        elif 'sso' in request.query_params:
```

Comment:
This endpoint is redundant if there is the fallback redirecting to the sso-endpoint anyways. Or did I miss something here?